app/styles/_overview.less

```diff
@@ -280,6 +280,11 @@
       }
     }
   }
+  .list-pf-uglyname {
```

Comment:
I should have mentioned this on first review, but suggest we avoid using the `list-pf-*` prefix ourselves since that is effectively Patternfly's namespace. Maybe `list-row-longname`